specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-04-01/webapplicationfirewall.json

```diff
@@ -1,636 +0,0 @@
-{
-  "swagger": "2.0",
-  "info": {
-    "version": "2018-08-01",
-    "title": "WebApplicationFirewallManagement",
```

Comment:
This old WAF API spec should not have been included with the 2019-04-01 version.